### PR TITLE
EAR-1374-update-delete-answer-icon-v2

### PR DESCRIPTION
**Pull request by MattElcock**
### What is the context of this PR?

The 'Create Questionnaire' modal had a trash can icon instead of a cross close icon in tthe top right.

### How to review

Make sure the icon is updated.

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process

### CODE REVIEW

**Comment by tbaggaley** [comment]
It is a little different to the existing styling on prod - is that intentional?

### This PR
Screenshot 2021-06-25 at 12 28 01

### Previous design (on prod)
Screenshot 2021-06-25 at 12 28 07

**Review by farres1 — DISMISSED** [review]

**Review by farres1 — COMMENTED** [review]
Adding on from what Tom said, the orange border box around the cross is also off centre when clicking the cross. The cross is at the top left corner of the box.

**Review by farres1 — APPROVED** [review]

File: eq-author/src/components/buttons/CloseButton/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const CloseButton = (props) => (
+  <StyledCloseButton role="button" {...props}>
+    <CloseIcon />
+  </StyledCloseButton>
+);
+
+CloseButton.defaultProps = {
+  size: "medium",
+  type: "button",
+  "aria-label": "Delete",
```
Comment (tbaggaley) [inline eq-author/src/components/buttons/CloseButton/index.js:68]:
```suggestion
  "aria-label": "Close",
```

Nitpicky, but considering it's a `CloseButton`... ;-)

File: eq-author/src/components/modals/Modal/index.js [hunk 2]
```diff
@@ -3,11 +3,11 @@ import ReactModal from "react-modal";
...
 import PropTypes from "prop-types";
 import { colors } from "constants/theme";
-import DeleteButton from "components/buttons/DeleteButton";
+import UnstyledCloseButton from "components/buttons/CloseButton";
 
 const ANIMATION_DURATION = 100;
 
-export const CloseButton = styled(DeleteButton).attrs({
+export const CloseButton = styled(UnstyledCloseButton).attrs({
   "aria-label": "Close",
```
Comment (tbaggaley) [inline eq-author/src/components/modals/Modal/index.js:11]:
```suggestion
```

**Review by tbaggaley — COMMENTED** [review]
Styling's looking good :-)

File: eq-author/src/components/buttons/CloseButton/index.js (hunk 1 above)
Comment (MattElcock) [reply eq-author/src/components/buttons/CloseButton/index.js:68]: Oh sorry, I missed that one!

**Review by tbaggaley — DISMISSED** [review]

**Review by farres1 — APPROVED** [review]

**Review by tbaggaley — APPROVED** [review]

**Review by farres1 — APPROVED** [review]

**Pull request merged**